.gitignore

```diff
@@ -101,3 +101,5 @@ version.ml
 version.texi
 .cargo
 !src/vendor/**
+**/target/rls/debug/
+/src/target/
```

Comment:
Note to self: These directories probably shouldn't exist, I should investigate why they show up before this gets merged and I stop seeing them in `git status`, because then I'll never remember.